include/arm/96boards.h

```diff
@@ -38,6 +38,7 @@ extern "C" {
 #define MRAA_96BOARDS_LS_SPI_COUNT  1
 #define MRAA_96BOARDS_LS_UART_COUNT 2
 #define MRAA_96BOARDS_LS_PIN_COUNT  40
+#define MRAA_96BOARDS_LED_COUNT	6
```

Comment:
Some whitespace error between define name and value - I guess that's a TAB symbol instead of space

Reply:
Ack